repairnator/src/main/java/fr/inria/spirals/repairnator/serializer/AbstractDataSerializer.java

```diff
@@ -60,6 +60,12 @@ protected String getPrettyPrintState(ProjectState state, GatherTestInformation g
             case PREVIOUSVERSIONISNOTINTERESTING:
                 return "previous build is not interesting";
 
+            case BUILDCHECKEDOUT:
```

Comment:
I'm not sure this is the only place you have to add new cases... did you check on google sheets serializer?